### DISCUSSION TITLE
Grafana Admin Login

### DISCUSSION THREAD

**Discussion by its-a-me-mario-the-og** [Security-Onion-Solutions/securityonion#8206] [Unsupported Versions] [answered]
Hi I have been trying to create alerts for my single node using Grafana. I went through the documentation and the youtube video created in January of this year. When I went to login as Admin and used the "sudo salt-call pillar.get secrets" I got the password and put it in. So far it has not worked and gives me an error message saying "Invalid username or password". I went back through the video and documentation to make sure I did everything right but it won't let me in. I am using the password under grafana_admin when I use the pillar.get secrets command. Thank you in advance for any help or suggestions.

**Comment by xfaith** [comment, marked as answer]
Admin is case sensitive, just want to check and make sure you are using admin, not Admin to login to the page.   I just revalidated login with the secrets grafana_admin on my setup and had no issues.
